includes/ui/class-ui.php

```diff
@@ -52,14 +60,39 @@ public function enqueue_styles() {
 
 		if ( $this->has_ui ) {
 			wp_enqueue_style( 'wp-job-manager-ui' );
+
+			if ( ! empty( $this->css_variables ) ) {
+				wp_add_inline_style( 'wp-job-manager-ui', $this->generate_inline_css() );
+			}
 		}
 	}
 
 	/**
 	 * Request the styles to be loaded for the page.
+	 *
+	 * @param array $css_variables An array of CSS variables to be enqueued: <variable_name> => <value>.
+	 */
+	public static function ensure_styles( $css_variables = null ) {
```

Comment:
```suggestion
	public static function ensure_styles( array $css_variables = [] ) {
```

If `$css_variables` becomes `null` for any reason, the `foreach` below will throw a warning, so better to fix that in the API. :)